web/src/features/charts/FuturePrice.cy.tsx

```diff
@@ -20,24 +20,16 @@ describe('FuturePrice only positive values', () => {
     cy.get('[data-test-id="future-price"]').should('not.exist');
     cy.get('[data-test-id="collapse-button"]').click();
     cy.get('[data-test-id="future-price"]').should('be.visible');
-  });
```

Comment:
For some unknown reason the tests fails when in their own it statements, but works when it's combined. 🤷🏻‍♂️

Reply:
Both! But I can't really tell why. It's working as a combined test though.

Reply:
🤔 Curious. I don't think it'll be a blocker if it works as a combined test. I might play around with it locally though

Reply:
So I branched this branch, reintroduced the tests, and it [worked](https://github.com/electricitymaps/electricitymaps-contrib/actions/runs/11164629034/job/31034519914?pr=7275)! Seems like you fixed whatever the issue was 🎉

Reply:
Maybe it had something to do with the [initial state](https://github.com/electricitymaps/electricitymaps-contrib/pull/7273)?

Reply:
Maybe?

But thanks for re-adding them!